k8s-plugin/src/main/kotlin/com/amazon/spinnaker/keel/k8s/resolver/CredentialsResourceHandler.kt

```diff
@@ -31,19 +32,23 @@ class CredentialsResourceHandler(
     private val encoder: Base64.Encoder = Base64.getMimeEncoder()
 
     public override suspend fun toResolvedType(resource: Resource<CredentialsResourceSpec>): K8sCredentialManifest {
-        if ((resource.spec.template.metadata[TYPE] as String?).isNullOrBlank()) {
-            throw CredResourceTypeMissing("missing \".metadata.type\" for the credential")
+        try {
+            require(!(resource.spec.template.metadata[TYPE] as String?).isNullOrEmpty()) {"missing \".metadata.type\" field for the credential"}
+            require(!(resource.spec.template.metadata[CLOUDDRIVER_ACCOUNT] as String?).isNullOrEmpty()) {"missing \".metadata.account\" field for the credential"}
```

Comment:
probably should get rid of `field` in the msg. The field could be there and be empty.

Reply:
I'll update the message to indicate it's either empty or absent.